main/.vitepress/config.mjs

```diff
@@ -83,6 +83,11 @@ export default defineConfig({
   ignoreDeadLinks: [
     // ignore all localhost links
     /^https?:\/\/localhost/,
+    // ignore links that starts with ./packages/*
```

Comment:
Some of the `.md` files generated from endo and agoric-sdk repo use these links below, which will fail the vitepress build as they're indeed dead links. There are 8 total offenders so it's not widespread, and I'm adding all of them here so they can be ignored by vitepress